src/libponyrt/asio/asio.c

```diff
@@ -42,6 +58,24 @@ void ponyint_asio_init(uint32_t cpu)
 {
   asio_cpu = cpu;
   running_base.backend = ponyint_asio_backend_init();
+
+#if defined(USE_SYSTEMATIC_TESTING)
+#if defined(PLATFORM_IS_WINDOWS)
+    // create wait event objects
+    running_base.sleep_object = CreateEvent(NULL, FALSE, FALSE, NULL);
+#elif defined(USE_SCHEDULER_SCALING_PTHREADS)
+    // TODO: memtrack accounting
```

Comment:
I'd like to see any TODO's be able to be easily found like...

TODO systematic testing:

Or some unique value and then opening an issue with a checklist of things to add/do that notes what those things are and notes the special todo value to look for.

Reply:
Replaced `TODO:` with `TODO systematic testing:`. Will open an issue with the checklist/string to search for once this PR is merged.